README.md

```diff
@@ -189,13 +189,13 @@ If successful, do a query again and balance of account1 should be 110 now.
 
 This tool is based on fabric-go-sdk. WASM pusher tool is an alternative to install wasm binary or compressed zip wasm chaincode directly instead of sending hex version using cli.
 
-More information here : [link](https://github.com/kleash/fabric-chaincode-wasm/tree/master/tools/wasm-pusher)
+More information here : [link](tools/wasm-pusher)
 
 ## Run Unit test
 
- - Compile sample rust chaincode at ```sample-wasm-chaincode/chaincode_example02/rust/src/lib.rs```  to wasm binary : [instructions](https://github.com/kleash/fabric-chaincode-wasm/tree/master/sample-wasm-chaincode/README.md)
- - Go to ```wasmcc``` directory
- - Give command ```go test```
+ - Compile sample rust chaincode at `sample-wasm-chaincode/chaincode_example02/rust/src/lib.rs`  to wasm binary : [instructions](sample-wasm-chaincode/README.md)
+ - Go to `wasmcc` directory
+ - Give command `go test`
```

Comment:
a TODO for later. should attempt to refer to the makefile.

Reply:
@MHBauer good point. opened an issue here for same: #25